app/spdb/build.sbt

```diff
@@ -221,32 +222,36 @@ def swalker(version: Version) = AppConfig(
   )
 ) extending List(with_gogo(version), swalker_credentials(version))
 
-// FNUSSBER
-def fnussber(version: Version) = AppConfig(
```

Comment:
I think olesja/sabrina still use florian's credentials. I'd check with them to create their own credentials